src/engine/experimental/eip7547.md

```diff
@@ -26,11 +26,13 @@ This specification is based on and extends [Engine API - Cancun](./cancun.md) sp
 
 ## Structures
 
+### InclusionListConfiguration
```

Comment:
unused atm, should be added to `NewInclusionListV1`

Reply:
i referenced the value "2. Client software **MUST** validate that the sum of inclusion list transactions gas does not exceed `inclusionListMaxGas`. " does it need to be a param to the `newILv1` function ? or is there a better way to specify globals ?

Reply:
I think it would be best to specify as a global

Reply:
agree, this should be a constant

Reply:
ok making a global!

Reply:
making a global in the EELS spec, not in the engine API